vizro-core/src/vizro/models/_action/_action.py

```diff
@@ -70,21 +70,21 @@ def _get_callback_mapping(self):
         """
         from vizro.actions._callback_mapping._get_action_callback_mapping import _get_action_callback_mapping
 
+        # call this callback_inputs_kwargs
         callback_inputs: Dict[str, Any] = {
             **_get_action_callback_mapping(action_id=ModelID(str(self.id)), argument="inputs"),
-            **{
-                f'{input.split(".")[0]}_{input.split(".")[1]}': State(input.split(".")[0], input.split(".")[1])
-                for input in self.inputs
-            },
             "trigger": Input({"type": "action_trigger", "action_name": self.id}, "data"),
         }
 
+        # something like this:
+        # callback_inputs_args = [State(input.split(".")[0], input.split(".")[1]) for input in self.inputs]
+
         callback_outputs: Dict[str, Any] = {
             **_get_action_callback_mapping(action_id=ModelID(str(self.id)), argument="outputs"),
             **{
                 f'{output.split(".")[0]}_{output.split(".")[1]}': Output(
                     output.split(".")[0], output.split(".")[1], allow_duplicate=True
-                )
+                )  # change to output: Output(...)
```

Comment:
@petar-qb These are the changes we were talking about to have `callback_input_args` and `callback_input_kwargs`.